includes/admin/post-types/meta-boxes/class.llms.meta.box.lesson.php

```diff
@@ -191,8 +160,67 @@ public function get_fields() {
 				),
 			),
 		);
+
+		if ( 'yes' !== $course->get( 'lesson_drip' ) || ! $course->get( 'drip_method' ) ) {
+			$fields['drip']['fields'][] = array(
+				'class'         => 'llms-select2',
+				'desc_class'    => 'd-all',
+				'id'            => $this->prefix . 'drip_method',
+				'is_controller' => true,
+				'label'         => __( 'Method', 'lifterlms' ),
+				'type'          => 'select',
+				'value'         => $methods,
+			);
+			$fields['drip']['fields'][] = array(
+				'controller'       => '#' . $this->prefix . 'drip_method',
+				'controller_value' => 'lesson,enrollment,start,prerequisite',
+				'class'            => 'input-full',
+				'id'               => $this->prefix . 'days_before_available',
+				'label'            => __( 'Delay (in days) ', 'lifterlms' ),
+				'type'             => 'number',
+				'step'             => 1,
+				'min'              => 0,
+			);
+			$fields['drip']['fields'][] = array(
+				'controller'       => '#' . $this->prefix . 'drip_method',
+				'controller_value' => 'date',
+				'class'            => 'llms-datepicker',
+				'id'               => $this->prefix . 'date_available',
+				'label'            => __( 'Date Available', 'lifterlms' ),
+				'type'             => 'date',
+			);
+			$fields['drip']['fields'][] = array(
+				'controller'       => '#' . $this->prefix . 'drip_method',
+				'controller_value' => 'date',
+				'class'            => '',
+				'desc'             => __( 'Optionally enter a time when the lesson should become available. If no time supplied, lesson will be available at 12:00 AM. Format must be HH:MM AM', 'lifterlms' ),
+				'id'               => $this->prefix . 'time_available',
+				'label'            => __( 'Time Available', 'lifterlms' ),
+				'type'             => 'text',
+			);
+		}
+
+		return $fields;
 	}
 
+	/**
+	 * Helpful messaging depending on whether the course for this lesson has drip settings enabled or not.
+	 *
+	 * @since [version]
+	 *
+	 * @param LLMS_Course $course Course object.
+	 * @return string
+	 */
+	public function get_drip_course_settings_info_html( $course ) {
+		$output = 'yes' === $course->get( 'lesson_drip' ) && $course->get( 'drip_method' ) ?
+			__( 'Drip settings are currently set at the course level, under the Restrictions settings tab. Disable to allow lesson level drip settings.', 'lifterlms' )
```

Comment:
```suggestion
			__( 'Drip settings are currently set at the course level, under the Restrictions settings tab. If you would like to set individual drip settings for each lesson, you must disable the course level drip settings first.', 'lifterlms' )
```
A bit clearer for now. If we get a lot of inquiries about this, it might help to show the course level settings in line here.